Orange/widgets/utils/itemmodels.py

```diff
@@ -646,6 +647,60 @@ def python_variable_tooltip(self, var):
         text += self.variable_labels_tooltip(var)
         return text
 
+
+class DomainModel(VariableListModel):
```

Comment:
Tests added and also put in separate commit las week (4c8a628800641ec6a89e315549e2d30b74233870).